web/views.py

```
@@ -5101,6 +5102,114 @@
     return redirect("course_detail", slug=slug)
 
 
+@login_required
+@teacher_required
+def send_nft_badge(request, achievement_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_nft_badge`

(ANN201)